components/new_channel_flow/new_channel_flow.tsx

```diff
@@ -192,7 +227,7 @@ export default class NewChannelFlow extends React.Component {
         }
     };
 
-    render() {
+    render = (): JSX.Element => {
```

Comment:
why the binding here?

Reply:
Sorry, by a mistake. I've just switched it to a normal function.